src/dashboards/teachers/common.libsonnet

```
@@ -0,0 +1,115 @@
+// Video commons
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local template = grafana.template;
+local common = import '../common.libsonnet';
+
+{
+  datasources: {
+    lrs: 'lrs',
+    edx: 'edx',
+  },
+  fields: {
+    context_extensions_completion_threshold: 'context.extensions.https://w3id.org/xapi/video/extensions/completion-threshold',
+    result_extensions_length: 'result.extensions.https://w3id.org/xapi/video/extensions/length',
+    result_extensions_time: 'result.extensions.https://w3id.org/xapi/video/extensions/time',
+    verb_display_en_us: 'verb.display.en-US.keyword',
+    video_id: 'object.id.keyword',
+  },
+  queries: {
+    video_id: 'object.id.keyword:$VIDEO',
```

Should we fix the `${FOO}` variable syntax convention?

```suggestion
    video_id: 'object.id.keyword:${VIDEO}',
```